packages/api/AlvTimeWebApi/Controllers/InvoiceRateController.cs

```diff
@@ -38,8 +41,8 @@ public async Task<decimal> FetchUserInvoiceRate(DateTime? fromDate, DateTime? to
 
     [HttpGet("InvoiceStatistics")]
     [Authorize(Policy = "AllowPersonalAccessToken")]
-    public async Task<InvoiceStatisticsDto> FetchUserInvoiceStatistics(DateTime fromDate, DateTime toDate)
+    public async Task<InvoiceStatisticsDto> FetchUserInvoiceStatistics(DateTime fromDate, DateTime toDate, InvoicePeriods period, ExtendPeriod extendPeriod, bool includeZeroPeriods)
```

Comment:
Adding a default parameter to InvoicePeriods, making it `InvoicePeriods period = InvoicePeriod.Month` and add the Labels-thing commented earlier and you will have a backend that does not break with any existing functionality :bulb:

Reply:
How about setting InvoicePeriod.Month to 0 in the enum instead?